docs/get-started/yarn-gpu.md

```diff
@@ -117,3 +117,41 @@ On all masters:
 ```bash
 sudo systemctl restart hadoop-yarn-resourcemanager.service
 ```
+
+Note: If `cgroup` is mounted on `tmpfs`, after one node rebooted,
+the cgroup directory permission get reverted, you can manually 
+update the cgroup permission:
+```bash
+chmod a+rwx -R /sys/fs/cgroup/cpu,cpuacct
+chmod a+rwx -R /sys/fs/cgroup/devices
+```
+or you can add the operation as init.d scripts:
```

Comment:
If we provide a sample of systemd service, we should modify here. It's not init.d.

Reply:
updated, thanks